src/Network/ManticoreClient/HTTPClient.php

```diff
@@ -91,19 +98,18 @@ public function sendRequest(string $request, ManticoreEndpoint $endpoint = null)
 		// Hmm, This is strange, but $endpoint === ManticoreEndpoint::Sql is not working
 		$prefix = ($endpoint->name === 'Sql' ? 'query=' : '');
 		$fullReqUrl = "{$this->url}/{$endpoint->value}";
-		$buddyVersion = buddy_version();
+		$agentHeader = $disableAgentHeader ? '' : "User-Agent: Manticore Buddy/{$this->buddyVersion}\n";
```

Comment:
Can you explain this magic?

Reply:
Yes, remember, I've discussed this with Stas in the chat. The purpose is that daemon would still be able to redirect some error requests from Buddy back to it. ( That's needed  for the processing of error `/cli` requests).  To let daemon do this, we must get off the Buddy's agent header.

Reply:
I did not get the case. Can you explain some examples of when we will get into this issue? Just to understand if we can find better solution for this.